cli/install/template.go

```diff
@@ -282,6 +282,105 @@ spec:
     kind: ServiceProfile
     shortNames:
     - sp
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+          - routes
+          properties:
+            routes:
+              type: array
+              items:
+                type: object
+                required:
+                - name
+                - condition
+                properties:
+                  name:
+                    type: string
+                  condition:
+                    type: object
+                    oneOf:
```

Comment:
I think it might be more clear and more correct to remove the oneOf and have `method`, `pathRegex`, `all`, `any`, and `not` be properties of `condition`.